src/vm/eventpipe.cpp

```diff
@@ -227,13 +222,20 @@ void EventPipe::Shutdown()
 {
     CONTRACTL
     {
-        THROWS;
+        NOTHROW;
         GC_TRIGGERS;
         MODE_ANY;
     }
     CONTRACTL_END;
 
-    Disable();
+    // EventPipe::Disable is additionally can throw exceptions
```

Comment:
Are you trying to say that EventPipe::Disable is allowed to throw exceptions?  I suspect you got interrupted when writing this.  :)